src/components/Auth/LandingFooter.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, memo } from 'react';
+
+import { useCurrentCriteriaContext } from 'components/CurrentCriteriaProvider';
+import mitre from 'assets/mitre.png';
+import mcode from 'assets/mcode.png';
+import useStyles from './styles';
+
+const LandingFooter: FC = () => {
+  const styles = useStyles();
+  const { resetCurrentCriteria } = useCurrentCriteriaContext();
+
+  return (
+    <footer className={styles.footer}>
+      <a
+        href="https://mitre.org/"
```

Comment:
This still hangs for me. I think it needs to be 
```suggestion
        href="https://www.mitre.org/"
```